web/installmanager/index.js

```diff
@@ -31,22 +60,44 @@
         nameContainer.appendChild(details);
 
         let goBtn = document.createElement('button');
-        goBtn.style.padding = '8px';
+        goBtn.style.padding = '4px';
         goBtn.style.textAlign = 'center';
         goBtn = adaptForIcons(goBtn);
-        goBtn.innerHTML = icon('sync_arrow_up', '18px') + ' Switch';
+        goBtn.innerHTML = icon('sync_arrow_up', '18px') + '';
```

Comment:
again with the concatenation of an empty string with another string, WHY
```suggestion
        goBtn.innerHTML = icon('sync_arrow_up', '18px');
```